Sources/Sentry/Profiling/SentryProfiledTracerConcurrency.mm

```diff
@@ -81,7 +80,7 @@
 _unsafe_cleanUpContinuousProfilerV2()
 {
     if (_gInFlightRootSpans == 0) {
-        SENTRY_TEST_FATAL(@"Attempted to decrement count of root spans to less than zero.");
+        SENTRY_LOG_ERROR(@"Attemtpted to stop continuous profiler with no root spans in flight.");
```

Comment:
@phil, once we know this works, I think we should add a comment explaining why we changed this, so we don't forget. I'm 99% sure, you are going to add this anywas, but still I wanted to put it here, so we don't forget.